maestro-orchestra/src/main/java/maestro/orchestra/Orchestra.kt

```diff
@@ -469,13 +465,8 @@ class Orchestra(
                         onCommandSkipped(index, command)
                         false
                     } catch (e: Throwable) {
-                        when (onCommandFailed(index, command, e)) {
-                            ErrorResolution.FAIL -> throw e
-                            ErrorResolution.CONTINUE -> {
-                                // Do nothing
-                                false
-                            }
-                        }
+                        onCommandFailed(index, command, e)
+                        throw e
```

Comment:
@dmitry-zaitsev do you know why `runSubFlow` throws here, while `executeCommands` does not?

Reply:
As discussed in person, I do not remember _exactly_ why it was needed but I believe there was a need for that. Something possibly related to CLI output or to special exception types that we need to propagate up (i.e. when command is skipped)